src/lib/ssr-api.ts

```diff
@@ -21,3 +20,19 @@ export const getEvents = async ({ tags }: Tags) => {
 
   return await getResourceCollection(NODE_TYPES.EVENT, { params: eventParams().getQueryObject() })
 }
+
+export const getNews = async (shortList: string) => {
+  const newsParams = () =>
+    baseArticlePageQueryParams()
+      .addSort('created', 'DESC')
+
+  const newsParamsLimited = () =>
+    baseArticlePageQueryParams()
+      .addSort('created', 'DESC')
+      .addPageLimit(4)
+
+  if (shortList === 'true'){
+    return await getResourceCollection(NODE_TYPES.ARTICLE, { params: newsParamsLimited().getQueryObject() })
+  }
+  return await getResourceCollection(NODE_TYPES.ARTICLE, { params: newsParams().getQueryObject() })
+}
```

Comment:
getNews function could be structured a bit differently. I moved the if clause and made some changes how jsonapi params are added:

```
export const getNews = async (shortList: string) => {
  const newsParams = () =>
    baseArticlePageQueryParams()
      .addSort('created', 'DESC')
  
  if (shortList === 'true'){
    const newsParamsLimited = () =>
      newsParams()
        .addPageLimit(4)
        .getQueryObject()

    return await getResourceCollection(NODE_TYPES.ARTICLE, { params: newsParamsLimited() })
  }

  return await getResourceCollection(NODE_TYPES.ARTICLE, { params: newsParams().getQueryObject() })
}
```